packager/app/packager_main.cc

```diff
@@ -106,7 +106,10 @@ const char kUsage[] =
     "    The group ID for the output stream. For HLS this is used as the\n"
     "    GROUP-ID attribute for EXT-X-MEDIA.\n"
     "  - playlist_name: Required for HLS output.\n"
-    "    Name of the playlist for the stream. Usually ends with '.m3u8'.\n";
+    "    Name of the playlist for the stream. Usually ends with '.m3u8'.\n"
+    "  - skip_encryption: Optional value which specifies if encryption of\n"
+    "    the stream shall be skipped. By default, if encryption is enabled,\n"
```

Comment:
I am fine with both. I am slightly leaning towards 0|1 for consistency since we also use 0|1 in udp options.

> I also have to mention: Thank you for the friendly and good feedback I've gotten from you during the last few days! :) I personally find it very encouraging!

No problem. Thanks for your work on improving shaka-packager :-)